src/pydocstyle/parser.py

```diff
@@ -218,6 +218,14 @@ def is_overload(self):
                 return True
         return False
 
+    @property
+    def is_property(self):
+        """Return True iff the method decorated with property."""
```

Comment:
This says `iff` which tends to only mean "if and only if" but that's not actually true here because anything with the string "property" in the name will return True, not just the `@property` built-in decorator